### ISSUE TITLE
Plugins incompatible with pytest 4, leads to ContextualVersionConflict

### ISSUE THREAD

**Issue by jaraco** [manahl/pytest-plugins#110]
As can be seen in [this commit](https://github.com/manahl/pytest-plugins/commit/061f2e8609ada52f39b21a00949335ac030c3515), none of the plugins are compatible with pytest. This causes test suites that depend on them to fail. Consider:

```
setuptools master $ pip-run -q pytest pytest-virtualenv -- -c "import pkg_resources; pkg_resources.require('pytest-virtualenv')"                                                         
pytest-shutil 1.4.0 has requirement pytest<4.0.0, but you'll have pytest 4.1.1 which is incompatible.
pytest-fixture-config 1.4.0 has requirement pytest<4.0.0, but you'll have pytest 4.1.1 which is incompatible.
pytest-virtualenv 1.4.0 has requirement pytest<4.0.0, but you'll have pytest 4.1.1 which is incompatible.
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/Users/jaraco/code/main/setuptools/pkg_resources/__init__.py", line 898, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/Users/jaraco/code/main/setuptools/pkg_resources/__init__.py", line 789, in resolve
    raise VersionConflict(dist, req).with_context(dependent_req)
pkg_resources.ContextualVersionConflict: (pytest 4.1.1 (/private/var/folders/c6/v7hnmq453xb6p2dbz1gqc6rr0000gn/T/pip-run-svi02ctd), Requirement.parse('pytest<4.0.0'), {'pytest-shutil', 'pytest-fixture-config', 'pytest-virtualenv'})
```

**Comment by jonathansp** [comment]
@javefang @eeaston any specific reason this plugin is not compatible with v4.x.x?

**Comment by eeaston** [comment]
Hi all, we just merged #134 to address this, thanks

**Comment by jvesely** [comment]
> Hi all, we just merged #134 to address this, thanks

Hi, do you plan a release that includes this change in the near future?

**Comment by jcfr** [comment]
@eeaston I think this issue can be closed.

**Comment by eeaston** [comment]
Released in 1.6.0 - many apologies all for the delay in releasing to pypi.
